src/components/FormInputField.vue

```diff
@@ -51,6 +54,10 @@ export default {
     type: {
       type: String,
       default: 'text'
+    },
+    autocomplete: {
+      type: String,
+      default: 'off'
```

Comment:
🤔 we shouldn't set it `off` by default, I think that regarding a11y, we should set this prop required, type String